pages/blog/index.tsx

```diff
@@ -35,18 +35,19 @@ export const getServerSideProps: GetServerSideProps = async context => {
   let tagTotalPosts;
   let filteredPosts = posts;
   let filteredTotalPosts = totalPosts;
+  let resultsPerPage = CMS.BLOG_RESULTS_PER_PAGE;
```

Comment:
This resultPerPAge and the one calculcated here `tag ? 12 : CMS.BLOG_RESULTS_PER_PAGE` are the.

You can put resultPerPAge at the start of the function, and reuse it. or not?